src/domain/interfaces/account.datasource.interface.ts

```diff
@@ -22,11 +25,11 @@ export interface IEmailDataSource {
    * @param args.safeAddress - the Safe address to use as filter
    * @param args.account - the owner address to which link the email address is linked to
    */
-  getEmail(args: {
+  getAccount(args: {
```

Comment:
I think this function documentation needs to be changed as well (from the concept `email` to the `account` one).

Reply:
[fa63a4e](https://github.com/safe-global/safe-client-gateway/pull/1050/commits/fa63a4e9950036c1b044c936774359b805c7c62f)